package.json

```diff
@@ -2,10 +2,15 @@
   "name": "@semantic-release/github",
   "description": "semantic-release plugin to publish a GitHub release and comment on released Pull Requests/Issues",
   "version": "0.0.0-development",
+  "type": "module",
   "author": "Pierre Vanduynslager (https://twitter.com/@pvdlg_)",
   "ava": {
     "files": [
       "test/**/*.test.js"
+    ],
+    "nodeArguments": [
+      "--loader=quibble",
+      "--no-warnings"
```

Comment:
what is the `no-warnings` flag?

Reply:
custom loaders are currently experimental. For every test file several lines of Node warnings are logged